src/Application.php

```diff
@@ -141,7 +146,7 @@ protected function initSabreDAV() {
                 $this['config']['auth']['type'],
                 $this['config']['auth']['realm'],
                 $this['pdo'],
-                null
+                '/dav.php'
```

Comment:
neither
![dav1](https://cloud.githubusercontent.com/assets/11665806/17960817/8a17ea82-6aa8-11e6-9460-23c38b2856d6.png)
nor
![dav2](https://cloud.githubusercontent.com/assets/11665806/17960822/8dee40b6-6aa8-11e6-9cf6-a18eb3f1db02.png)
works for me...
:-(
What am I doing wrong?